airflow/operators/bash.py

```diff
@@ -140,15 +141,20 @@ def __init__(
         env: dict[str, str] | None = None,
         append_env: bool = False,
         output_encoding: str = "utf-8",
-        skip_exit_code: int = 99,
+        skip_exit_code: int | None = None,
+        skip_on_exit_code: int = 99,
```

Comment:
I wonder if should set `None` as default value for `skip_on_exit_code` too, like by default the operator doesn't skip for any exit code.

Reply:
Changing default is a breaking change so we can't :(